timetable/utils.py

```diff
@@ -41,26 +41,26 @@ def get_timetable_context(lessons):
 
     lessons = lessons.select_related('teacher', 'group', 'room', 'subject')
 
+    # TODO: a cleaner way to pass period str to the template while using
+    #       period number as key?
     periods = [period.number for period in default_periods]
     period_strs = get_period_strings(default_periods)
 
-    alternative_periods = Period.objects.exclude(schedule=1) # Exclude default periods
-
-    # TODO: a cleaner way to pass period str to the template while using
-    #       period number as key?
-    alternative_periods = Period.objects.exclude(schedule=1) # Exclude default periods
-    alternative_day_plan = None
-    for day_plan in DayPlan.objects.all():
-        if day_plan.is_today:
-            alternative_day_plan = day_plan
-            break
+    try:
+        alternative_day_plan = DayPlan.objects.get(date=date.today())
+        alternative_periods = Period.objects.filter(schedule=alternative_day_plan.schedule)
+    except ObjectDoesNotExist:
+        alternative_day_plan = None
 
```

Comment:
Before geting the alternative periods, you should check if the alternative schedule is not the default one.
![image](https://github.com/zorzalo3/zorza/assets/44808028/f6d532ef-0fe3-4f66-adc3-7f12902fcfef)
![image](https://github.com/zorzalo3/zorza/assets/44808028/8aff8c82-7639-4527-8903-51c606c71104)